.github/workflows/maven.yaml

```diff
@@ -76,4 +76,4 @@ jobs:
       - name: Test Cli
         run: |
           cd jrd
-          mvn --batch-mode test -Dtest=CliTest -DfailIfNoTests=false
+          mvn --batch-mode test -Dtest=*CliTest -DfailIfNoTests=false
```

Comment:
Why there is this condition at all? Test all!
Also the -DfailIfNoTests=false seems wrong. lets fail